app/Http/Controllers/FaqController.php

```diff
@@ -26,11 +26,11 @@ public function index(){
         ]);
     }
 
-    public function create(){
+    public function create() {
         return view('faq.create');
     }
 
-    public function store(Request $request){
+    public function store(Request $request) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->